src/backend/controllers/tankController.js

```diff
@@ -1,68 +1,64 @@
 // @flow strict
 
-// Flow type import
-import type { Request, Response } from 'express';
-
 const User = require('../../models/userModel');
 const Tank = require('../../models/tankModel');
 
 const { validationResult } = require('express-validator');
 
-exports.getFavorite = async (req: Request, res: Response) => {
+// Flow type import
+import type {
+	$Request,
+	$Response,
+	NextFunction,
+	Middleware,
+} from 'express';
+
+exports.getFavorite = async (req: $Request, res: $Response) => {
```

Comment:
I guess I might be out of the loop when it comes to it, but what is the significance of `$Request/$Response` vs `Request/Response`? If this is about defining types for Flow, the latter ones are listed rather than the former.

Reply:
+1 to using Request/Response instead of $Request/$Response unless there is a very compelling reason not to